### ISSUE TITLE
Unable to start Raiden because of failed dependency

### ISSUE THREAD

**Issue by pcppcp** [raiden-network/raiden#2417]
```
git rev-parse HEAD
75f6d27d78016abda226536298b511c7d1d5f151
```

clean venv, fresh reinstall of `requirements-dev.txt`. Python 3.6.0

```
Traceback (most recent call last):
  File "/home/xoza/.pyenv/versions/3.6.0/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/xoza/.pyenv/versions/3.6.0/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/srv/data/src/raiden/raiden/__main__.py", line 15, in <module>
    main()
  File "/srv/data/src/raiden/raiden/__main__.py", line 11, in main
    run(auto_envvar_prefix='RAIDEN')  # pylint: disable=no-value-for-parameter
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/click/core.py", line 1043, in invoke
    return Command.invoke(self, ctx)
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/srv/data/src/raiden/raiden/ui/cli.py", line 1104, in run
    NodeRunner(kwargs, ctx).run()
  File "/srv/data/src/raiden/raiden/ui/cli.py", line 885, in run
    click.secho(self._welcome_string, fg='green')
  File "/srv/data/src/raiden/raiden/ui/cli.py", line 874, in _welcome_string
    return f"Welcome to Raiden, version {get_system_spec()['raiden']}!"
  File "/srv/data/src/raiden/raiden/utils/__init__.py", line 192, in get_system_spec
    raiden=pkg_resources.require(raiden.__name__)[0].version,
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/pkg_resources/__init__.py", line 967, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/xoza/.pyenv/versions/360_raiden/lib/python3.6/site-packages/pkg_resources/__init__.py", line 853, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pypiwin32>=223' distribution was not found and is required by web3
```

**Comment by LefterisJP** [comment]
@pcppcp is this issues related to https://github.com/raiden-network/raiden/issues/2418?

**Comment by CosminNechifor** [comment]
@pcppcp Fixed by: #2428
